routes/ember.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var getExamples    = require('../lib/examples').get;
+var renderExamples = require('../lib/examples').render;
+var pkgMeta        = require('../lib/package-meta')('ember-intl');
+
+module.exports = function (route) {
+    route.name = 'ember';
+
+    route.get(function (req, res, next) {
+        var isProduction = req.app.get('env') === 'production';
+
+        getExamples('ember', {cache: isProduction})
+            .then(function (examples) {
+                res.expose(examples, 'examples');
+                res.expose('integration', 'pageType');
+
+                res.render('ember', {
+                    activeMenuItem: route.name,
+                    package       : pkgMeta,
+                    examples      : renderExamples(examples, res.intl)
```

Comment:
All of the examples are still rendered on the server.

https://www.dropbox.com/s/897k925g050iefg/Screenshot%202015-02-02%2015.35.38.png?dl=0

I could be doing this react thing wrong..